.buildkite/commands/run-unit-tests.sh

```diff
@@ -11,5 +11,26 @@ gem install bundler
 echo "--- :rubygems: Setting up Gems"
 install_gems
 
-echo "--- 🧪 Testing"
+echo "--- 🔬 Testing"
+set +e
 bundle exec fastlane test_without_building name:WordPressUnitTests try_count:3
+TESTS_EXIT_STATUS=$?
+set -e
+
+if [[ "$TESTS_EXIT_STATUS" -ne 0 ]]; then
+  # Keep the (otherwise collapsed) current "Testing" section open in Buildkite logs on error. See https://buildkite.com/docs/pipelines/managing-log-output#collapsing-output
+  echo "^^^ +++"
+  echo "Unit Tests failed!"
+fi
+
+echo "--- 📦 Zipping test results"
+cd build/results/ && zip -rq WordPress.xcresult.zip WordPress.xcresult
+
+echo "--- 🚦 Report Tests Exit Status"
+if [[ $TESTS_EXIT_STATUS -eq 0 ]]; then
+  echo "Unit Tests seems to have passed (exit code 0). All good 👍"
+else
+  echo "The Unit Tests, ran during the '🔬 Testing' step above, have failed."
+  echo "For more details about the failed tests, check the logs under the '🔬 Testing' section and the \`.xcresult\` and test reports in Buildkite artefacts."
```

Comment:
Nitpick. While "artefacts" is a correct spelling, Buildkite uses the US English spelling:

```suggestion
  echo "For more details about the failed tests, check the logs under the '🔬 Testing' section and the \`.xcresult\` and test reports in Buildkite artifacts."
```

![image](https://user-images.githubusercontent.com/1218433/155070436-4739996c-04a6-4994-b5d9-a13f20703b44.png)

Reply:
Good point. I think I fell into the same typo trap in `run_ui_tests.sh` then. 🙃